helpers/tokens.test.js

```diff
@@ -0,0 +1,36 @@
+const jwt = require('jsonwebtoken');
+const { createToken } = require('./tokens');
+const { SECRET_KEY } = require('../config');
+
+describe('createToken', function() {
+	test('works: not admin', function() {
```

Comment:
```suggestion
	test('returns isAdmin as false if the user is not an admin', function() {
```

When reading the test cases, normally other developers would like to have a more understandable description. In your case, it works, but you will have to read the code first before knowing that it does

Reply:
I updated all the test descriptions to be more clear.